bump_version.sh

```diff
@@ -13,6 +13,17 @@ if [ -n "$NEW_VERSION" ]; then
   echo "Version bumped to $NEW_VERSION"
 fi
 
+#
+# Update the version in the podspec file
+#
 ./gradlew podspec
+
+# Remove the warning step about ./gradlew :generateDummyFramework, since on iOS we use CocoaPods to 
+# download the source code and build locally.
+PODSPEC_FILE="v4_abacus.podspec"
+
+# replace the line "if !Dir.exist?('build/cocoapods/framework/Abacus.framework') || Dir.empty?('build/cocoapods/framework/Abacus.framework')" with "if false"
+sed -i '' "s/if !Dir.exist?('build\/cocoapods\/framework\/Abacus.framework') || Dir.empty?('build\/cocoapods\/framework\/Abacus.framework')/if false/" $PODSPEC_FILE
+
 cd integration/iOS
-pod install
\ No newline at end of file
+pod install
```

Comment:
Do we need this iOS integration folder? Is it useful / up-to-date?

Reply:
Yes, we used that every once a while to test the integration.